### ISSUE TITLE
Error importing settings for 'CoffeeScript'

### ISSUE THREAD

**Issue by maurcz** [dracula/visual-studio#5]
Hey guys! Thank you for the awesome work with this theme.

 I got this message while installing Dracula in VS 2017 Enterpise: 

`Error 1:  An error occurred while importing settings for 'CoffeeScript' [code 5632].`

I've followed the installation instructions described [here](https://draculatheme.com/visual-studio/).

Screenshot of coffeescript file:

![coffee](https://user-images.githubusercontent.com/1469466/34416363-26a02578-ebc1-11e7-9248-f72ac6016a44.PNG)

(not sure if this is the highlight I should expect for coffeescript files)

Thanks!

**Comment by CervEdin** [comment]
Seems like I'm experiencing a similar error

`An error occurred while importing settings for 'CoffeeScript' [code 5297].`

VS2017 Professional 15.9.18

**Comment by epidemicz** [comment]
Was not able to reproduce the error in Visual Studio 2019 - I'm sure that has more to do with Visual Studio and how you're having it handle CoffeeScript, which is not natively supported.

I installed Web Essentials 2019 and I was able to handle .coffee files then and I noticed similar highlighting to yours.  I looked for coffee script syntax color bindings, but I was not able to find any. 

If a coffeescript expert comes along with a solution I'll be happy to work in whatever they can come up with, but I think we're running into a Visual Studio limitation at the moment.
